src/Uno.UI.Toolkit/UIElementExtensions.cs

```diff
@@ -55,7 +55,7 @@ public static double GetElevation(this UIElement element)
 			}
 		}
 
-#if __IOS__
+#if __IOS__ || __MACOS__
 		internal static void SetElevationInternal(this DependencyObject element, double elevation, Color shadowColor, CGPath path = null)
```

Comment:
I know it wasn't you @carldebilly , but it's bizarre that this method is not in a partial class def.

Reply:
I agree 100%.